src/main/java/com/google/cloud/genomics/dataflow/pipelines/VariantSimilarity.java

```diff
@@ -46,21 +55,29 @@
  */
 public class VariantSimilarity {
   private static final String VARIANT_FIELDS
-      = "nextPageToken,variants(start,calls(genotype,callSetName))";
+    = "nextPageToken,variants(start,calls(genotype,callSetName))";
+
+  public static interface VariantSimilarityOptions extends GenomicsDatasetOptions, GCSOptions {
+    @Default.Boolean(true)
+    Boolean getUseAlpn();
```

Comment:
oh, right, that thing! Yes, this definitely needs a comment. It could be a detailed comment in one place, and the other place just points to it (following the don't-repeat-yourself principle).